lib/ui/painting.dart

```diff
@@ -1591,6 +1607,21 @@ enum ImageByteFormat {
   /// image may use a single 8-bit channel for each pixel.
   rawUnmodified,
 
+  /// Raw extended range RGBA format.
+  ///
+  /// Unencoded bytes, in RGBA row-primary form with straight alpha, 32 bit
+  /// float (IEEE 754) per channel.
```

Comment:
iirc this is `IEEE754 binary32`, specifically?

Reply:
Yep, that's right.  There is another 32bit float format in that spec, decimal32.  Updated.